### PR TITLE
Add designer onboarding checklist

### PR DESCRIPTION
**Pull request by AlicjaSuska**
Adding a design-team-specific onboarding items after consulting with @jjinnii

### CODE REVIEW

**Comment by christinaforney** [comment]
We should make sure to add this to Process St too! @kyfli can help.

File: handbook/product/design/onboarding/index.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+Welcome to the Design Team!
+
+Havng gone though [the general onboarding](https://about.sourcegraph.com/handbook/people-ops/onboarding/general_onboarding) and [the Product onboarding](https://about.sourcegraph.com/handbook/product/onboarding), it's time for the Design onboarding!
+
+Your perspective as both a new user of Sourcegraph and a new teammate is very valuable to us. Please keep notes on any issues you encounter as you are onboarding and learning the product. We’ll use those notes to improve the product and process.
+
+## Designer checklist
+
+- Add your name to the [Project card](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1%3A203) designer list
+- Create your first Figma project
```
Comment (jjinnii) [inline handbook/product/design/onboarding/index.md:12]: The new designer may not be actively working on a feature or project during the first week or two -- Did we just want to leave this as "Get familiar with Project Tools, Templates, and Wildcard"?

Comment (AlicjaSuska) [reply handbook/product/design/onboarding/index.md]: I've moved the 'first figma project' to the separate section so it's not a part of the checklist but an independent item

**Comment by quinnkeast** [comment]
Should there be a step specifically about reading the product design team handbook content? I mean, I assume new folks will already be viewing that content, but it'd be good to call out.

File: handbook/product/design/onboarding/index.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+  - analytics-review
+  - wildcard-components-library-wcl
+  - hallway-testing
+- Make sure that your Name is set to your First and Last name on GitHub so that other teammates can easily indentify you. Go to GitHub -> Your profile -> Edit profile -> Fill out the 'Name field' -> Save
+- Add yourself as a designer to the handbook pages of the teams you're working with. You can also add those teams to your Slack profile description, for ex. 'Product Designer (Code Insights)'
+- Get familiar with the products that we use as inspirations for reference in the developer tools industry:
+  - code hosts: [GitHub](https://github.com/) and [GitLab] (https://gitlab.com/)
+  - code editors: [VS Code](https://code.visualstudio.com/) (and other IDEs like [IntelliJ](https://www.jetbrains.com/idea/) and [Atom](https://atom.io/))
+  - similar products: [grepp.app](https://grep.app/)
+
```
Comment (rrhyne) [inline handbook/product/design/onboarding/index.md:25]:
```suggestion
- Add yourself to the design club meeting on Tuesdays
```

**Comment by jjinnii** [comment]
@AlicjaSuska Can we try to merge this within the first few days of Marisa joining so she can have access? :)

File: handbook/product/design/onboarding/index.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
 # Onboarding
 
-(Coming soon!)
+Welcome to the Design Team!
+
+Havng gone though [the general onboarding](https://about.sourcegraph.com/handbook/people-ops/onboarding/general_onboarding) and [the Product onboarding](https://about.sourcegraph.com/handbook/product/onboarding), it's time for the Design onboarding!
```
Comment (quinnkeast) [inline handbook/product/design/onboarding/index.md:5]:
```suggestion
Having gone though [the general onboarding](https://about.sourcegraph.com/handbook/people-ops/onboarding/general_onboarding) and [the Product onboarding](https://about.sourcegraph.com/handbook/product/onboarding), it's time for the Product Design onboarding!
```

**Review by quinnkeast — APPROVED** [review]

**Comment by AlicjaSuska** [comment]
Closing with the [#90](https://github.com/sourcegraph/handbook/pull/90) merged to new handbook version. Thank you all for reviewing!

**Pull request closed without merging**